tests/scripts/test_spark_fail.sh

```diff
@@ -2,4 +2,4 @@
 # testing that job failure works properly w/ the operator
 # by throwing an exit code
 
-python -c 'import sys; sys.exit(1)'
\ No newline at end of file
+python -c "import sys; sys.exit(1)"
```

Comment:
So lack of newline is not an issue?

Reply:
flake8 and black mostly affects python files, so they do restrict the affects they can have other types of files.

Reply:
reran pre-commit gobally to catch it.